platform/android/os_android.cpp

```diff
@@ -477,6 +477,30 @@ void OS_Android::process_touch(int p_what, int p_pointer, const Vector<TouchPos>
 	}
 }
 
+void OS_Android::process_hover(int p_type, Point2 p_pos) {
+
+	switch (p_type) {
+		case 7: { //hover move
```

Comment:
Style wise, you can remove the `{ }` here to make it clear that it falls through `case 9`.

I guess this uses magic numbers because the constants come from the Java API? Could you add their original name in comment, and ideally a link to the relevant doc/code to find those values?